lib/pipeline.js

```diff
@@ -88,6 +88,23 @@ class PipelineModel extends BaseModel {
 
         return result.toLowerCase();
     }
+
+    /**
+     * Given a pipeline id, fetch the build admin
+     * @method getAdmin
+     * @param  {String}       pipelineId    Identifier for the Pipeline
+     * @param  {Function}     callback      fn(error, admin) where admin is the admin of the pipeline
+    */
+   // TODO: Retry if admin access was revoked
+    getAdmin(pipelineId, callback) {
```

Comment:
should this be getAdmins if we plan on retrying in the case where admin access was revoked?

Whoever consuming this should be doing the iteration, and not the function in the pipeline model